src/models/lineChart.js

```diff
@@ -251,7 +254,7 @@ nv.models.lineChart = function() {
                     .position({left: e.mouseX + margin.left, top: e.mouseY + margin.top})
                     .chartContainer(that.parentNode)
                     .valueFormatter(function(d,i) {
-                        return yAxis.tickFormat()(d);
+                        return d == null ? "N/A" : yAxis.tickFormat()(d);
```

Comment:
Well by default, the formatter will convert a 'null' to a 0, so you could argue that's the default behavior. Users can override that to their liking.